tests/pytests/requirements.txt

```diff
@@ -5,3 +5,4 @@ conan
 RLTest >= 0.7.11
 numpy >= 1.21.6
 scipy >= 1.7.3
+bfloat16 >= 1.0.1

```

Comment:
consider updating req versions to align with vecsim

Reply:
I prefer not to. Why did we change it in the library? What could fail if we don't?

Reply:
i used bfloat 1.1 and it didn't work with this version of numpy

Reply:
Here we actually install the latest version available (using `>=` and not `^`), so this way we let OSs with lower "latest" versions the option to install the latest compatible one. Let's keep it low unless even `1.0` doesn't support this version, or the merge queue rejects it?